bazel/utils/diff_test.bzl

```diff
@@ -185,6 +185,40 @@ diff_test = rule(
     test = True,
 )
 
+def md5sum_diff_test(name, srcs, expected = None, **kwargs):
+    """md5sum_diff_test is a diff_test that diffs the md5 checksums
```

Comment:
nit: we typically have a one line docstring, self contained, period at the end.

Followed by longer description.

Followed by `Args:`

With parameters simply indented, no * at the beginning?